source/css/_common/components/footer/footer.styl

```diff
@@ -7,9 +7,20 @@
 
 .footer-inner { text-align: center; }
 
+@keyframes heartAnimate {
+    0%,100%{transform:scale(1);}
+    10%,30%{transform:scale(0.9);}
+    20%,40%,60%,80%{transform:scale(1.1);}
+    50%,70%{transform:scale(1.1);}
+}
+#heart {
+    animation: heartAnimate 1.33s ease-in-out infinite;
+}
+
 .with-love {
   display: inline-block;
   margin: 0 5px;
+  color: rgb(255, 113, 168);
```

Comment:
I think need just to remove any color.

Reply:
Or like `border-radius` offer a custom define...haha

Reply:
Maybe, yeah. Why not?

Reply:
https://github.com/theme-next/hexo-theme-next/pull/195#issuecomment-374860647

Reply:
I don't know how to get rgb string...
i just remove the color setting?

Reply:
About color — this is another talk.

I'm talking for now about `animated` option: to turn off/on it. So, i think by default it must set on false, but if anybody want, he can to enable this option and icon will animate.

Reply:
Add `animated` under icon need to refactor the `footer.icon`, so i just add paralleling setting.